Apps/Sandcastle/gallery/Billboards.html

```diff
@@ -144,8 +144,8 @@
 function offsetByDistance() {
     Sandcastle.declare(offsetByDistance);
     Cesium.when.all([
-         Cesium.loadImage('../images/Cesium_Logo_overlay.png'),
-         Cesium.loadImage('../images/facility.gif')
+         Cesium.Resource.createIfNeeded('../images/Cesium_Logo_overlay.png').fetchImage(),
```

Comment:
`createIfNeeded` is private, so it shouldn't be used in Sandcastle examples.  To keep these as one-liners, the better approach would be to have a static `Resource.fetchImage` that just takes the same parameters as the Resource constructor and just returns a promise.

Reply:
Same comment applies anywhere we create a resource just to load and then throw it away.  I originally thought this could wait for a future PR, but I think these examples will get overly verbose if we don't do it now.